internal/app/proxyd/internal/frontend/frontend.go

```diff
@@ -154,6 +155,20 @@ func (r *ReverseProxy) Watch() (err error) {
 		return err
 	}
 
+	// Discovey local non loopback ips
+	ips, err := pkgnet.IPAddrs()
+	if err != nil {
+		log.Fatalf("failed to get local address: %v", err)
+	}
+	if len(ips) == 0 {
+		log.Fatalf("no IP address found for bootstrap backend")
+	}
+	ip := ips[0]
+
+	// Overwrite defined host so we can target local apiserver
+	// and bypass the admin.conf host which is configured for proxyd
+	config.Host = ip.String() + ":6443"
```

Comment:
dumb question, can we proxy to "127.0.0.1", or it doesn't listen on localhost?

Reply:
It should listen on localhost, but I believe that'd mean we need to run as a privileged container ( or at least with host namespace ) which I'm not sure proxyd does.

Reply:
thanks, got it!